docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,375 @@
 
 Note that when `rho_sample` and `rho_bead` are omitted, values for water and polystyrene are used for the sample and bead density respectively.
 
-Additionally, when the parameter `distance_to_surface` is omitted, a simpler model is used which assumes the experiment was performed deep in bulk (neglecting the increased drag induced by the nearby surface).
-
-Faxen's law
------------
-
-The hydrodynamically correct model presented in the previous section works well when the bead center is at least 1.5 times the radius above the surface.
+.. note::
 
-When going closer, the drag effect becomes stronger than the frequency dependent effects and better models to approximate the local drag exist.
+    One thing to note is that when using the hydrodynamically correct model, the equation for the drag does include the viscosity and bead diameter.
+    However, they now appear in a term which already amounts to a small correction therefore the impact of any errors in these is reduced :cite:`tolic2006calibration`.
 
-For lateral calibration, the following approximation is typically used :cite:`schaffer2007surface`:
+Calibration near the surface
+----------------------------
 
-.. math::
+So far, we have only considered experiments performed deep in bulk.
+In reality, proximity of the flowcell surface to the bead leads to an increase in the drag force on the bead.
+Such surface effects can be taken into account by specifying a distance to the surface (in microns)::
 
-    \gamma_\mathrm{faxen}(R/l) = \frac{\gamma_0}{
-        1 - \frac{9R}{16l} + \frac{1R^3}{8l^3} - \frac{45R^4}{256l^4} - \frac{1R^5}{16l^5}
-    }
+    force_model = lk.PassiveCalibrationModel(bead_diameter, hydrodynamically_correct=True, rho_sample=999, rho_bead=1060.0, distance_to_surface=5)
 
+As we approach the surface, the drag effect becomes stronger.
+The hydrodynamically correct model is only valid up to a certain distance to the surface.
+Moving closer, the frequency dependent effects become smaller than the overall drag effect and better models to approximate the local drag exist.
 We can use this model by setting `hydrodynamically_correct` to `False`, while still providing a distance to the surface::
 
-    force_model = lk.PassiveCalibrationModel(bead_diameter, hydrodynamically_correct=False, distance_to_surface=1e-6)
-
-Note that `pylake` always returns the bulk drag coefficient :math:`\gamma_0`.
+    force_model = lk.PassiveCalibrationModel(bead_diameter, hydrodynamically_correct=False, distance_to_surface=5)
 
-Axial Calibration
------------------
+To summarize, the workflow can be visualized as follows:
 
-For calibration in the axial direction, no hydrodynamically correct theory exists.
-In this case, one should use a Lorentzian with a specific correction term :cite:`schaffer2007surface`:
+.. image:: figures/force_calibration/surface_calibration_workflow.png
```

Comment:
I think a visual representation of the of the bead/surface would be good to explain which method to use in which situation, something like this
<img width="839" alt="Screenshot 2023-02-28 at 12 06 45" src="https://user-images.githubusercontent.com/17353250/221836339-bdfc823d-366c-4bd3-a697-c01fb1ef49ee.png">
We could also use it in the bluelake docs.

Reply:
![hydro-guide](https://user-images.githubusercontent.com/19836026/222182776-6d4e4bb7-63b7-4e2a-96bc-bdcb7434d951.jpg)
Courtesy of Thana :)